pkg/kubectl/cmd/cmd_test.go

```diff
@@ -209,7 +213,12 @@ type testFactory struct {
 	Err          error
 }
 
-func NewTestFactory() (*cmdutil.Factory, *testFactory, runtime.Codec, runtime.NegotiatedSerializer) {
+type fakeFactory struct {
+	tf    *testFactory
```

Comment:
@soltysh a general fakeFactory with all methods implemented with empty bodies? what if several commands want to use a same implementation of some methods?